ALVRTrackingInterface/ALVRTrackingInterface.cs

```diff
@@ -342,149 +366,152 @@ private void UpdateEye()
-                LookDown: (expressions[(int)FBExpression.Eyes_Look_Down_L] + expressions[(int)FBExpression.Eyes_Look_Down_R]) / 2.0f,
-                Openness: Math.Min(1, (opennessL + opennessR) / 2.0f),
-                Squeeze: Math.Min(1, (squeezeL + squeezeR) / 2.0f),
-                Widen: Math.Min(1, (eyeWidenL + eyeWidenR) / 2.0f)
+                Openness: Math.Min(1, eye.Right.Openness)
             );
 
             // Eye dilation code, automated process maybe?
-            UnifiedTrackingData.LatestEyeData.EyesDilation = 0.73f;
-            UnifiedTrackingData.LatestEyeData.EyesPupilDiameter = 0.0035f;
+            eye.Left.PupilDiameter_MM  = 0.0035f;
+            eye.Right.PupilDiameter_MM = 0.0035f;
+
+            #endregion
+        }
+
+        private void UpdateEyeExpressions(ref UnifiedExpressionShape[] unifiedExpressions, ref float[] expressions)
+        {
+            #region Eye Expressions Set
 
+            unifiedExpressions[(int)UnifiedExpressions.EyeWideLeft].Weight = expressions[(int)FBExpression.Upper_Lid_Raiser_L] * TrackingSensitivity.EyeWiden;
+            unifiedExpressions[(int)UnifiedExpressions.EyeWideRight].Weight = expressions[(int)FBExpression.Upper_Lid_Raiser_R] * TrackingSensitivity.EyeWiden;
 
+            unifiedExpressions[(int)UnifiedExpressions.EyeSquintLeft].Weight = expressions[(int)FBExpression.Lid_Tightener_L] * TrackingSensitivity.EyeSquint;
+            unifiedExpressions[(int)UnifiedExpressions.EyeSquintRight].Weight = expressions[(int)FBExpression.Lid_Tightener_R] * TrackingSensitivity.EyeSquint;
+
+            #endregion
+
+            #region Brow Expressions Set
+
+            unifiedExpressions[(int)UnifiedExpressions.BrowInnerUpLeft].Weight = expressions[(int)FBExpression.Inner_Brow_Raiser_L] * TrackingSensitivity.BrowInnerUp;
+            unifiedExpressions[(int)UnifiedExpressions.BrowInnerUpRight].Weight = expressions[(int)FBExpression.Inner_Brow_Raiser_R] * TrackingSensitivity.BrowInnerUp;
+            unifiedExpressions[(int)UnifiedExpressions.BrowOuterUpLeft].Weight = expressions[(int)FBExpression.Outer_Brow_Raiser_L] * TrackingSensitivity.BrowOuterUp;
+            unifiedExpressions[(int)UnifiedExpressions.BrowOuterUpRight].Weight = expressions[(int)FBExpression.Outer_Brow_Raiser_R] * TrackingSensitivity.BrowOuterUp;
+
+            unifiedExpressions[(int)UnifiedExpressions.BrowOuterDownLeft].Weight = expressions[(int)FBExpression.Brow_Lowerer_L] * TrackingSensitivity.BrowDown;
+            unifiedExpressions[(int)UnifiedExpressions.BrowInnerDownLeft].Weight = expressions[(int)FBExpression.Brow_Lowerer_L] * TrackingSensitivity.BrowDown;
+            unifiedExpressions[(int)UnifiedExpressions.BrowOuterDownRight].Weight = expressions[(int)FBExpression.Brow_Lowerer_R] * TrackingSensitivity.BrowDown;
+            unifiedExpressions[(int)UnifiedExpressions.BrowInnerDownRight].Weight = expressions[(int)FBExpression.Brow_Lowerer_R] * TrackingSensitivity.BrowDown;
```

Comment:
Where should the combined parameters be for external modules? There was some combined brow parameters that were in this module. Will this be internal for Unified Expressions?

Reply:
I believe we should only allow modules access to the base Unified Expression shapes, which modules have free-reign to send any transformed data they want to. All transforms of the base expressions into their combined counterparts are handled internally. My approach with the combined parameters is to try to retain the fidelity of the data and not to transmute the data like we were doing with V1 (SRanipal) combined parameters.

This architecture also lets us have a lot more flexibility in segregating modules from the internals of VRCFT.

Reply:
Need to add add the combined parameters
- BrowDownLeft
- BrowDownRight
- BrowsDown
- BrowsInnerUp
- BrowsOuterUp